tools/llm_bench/llm_bench_utils/model_utils.py

```diff
@@ -10,6 +10,20 @@
 import librosa
 
 
+KNOWN_PRECISIONS = [
+        'FP32', 'FP16',
+        'FP16-INT8', 'INT8', 'INT8_compressed_weights', 'INT8_quantized', 'PT_compressed_weights',
+        'OV_FP32-INT8', 'OV_FP16-INT8',
+        'OV_FP32-INT8_ASYM', 'OV_FP32-INT8_SYM', 'OV_FP16-INT8_ASYM', 'OV_FP16-INT8_SYM', 'OV_FP16-INT8_ASYM_HYBRID',
+        'PT_FP32-INT8', 'PT_FP16-INT8', 'PT_FP32-INT8_ASYM', 'PT_FP32-INT8_SYM', 'PT_FP16-INT8_ASYM', 'PT_FP16-INT8_SYM',
+        'GPTQ_INT4-FP32', 'GPTQ_INT4-FP16', 'INT4',
+        'OV_FP16-INT4_SYM', 'OV_FP16-INT4_ASYM', 'OV_FP32-INT4_SYM', 'OV_FP32-INT4_ASYM',
+        'OV_FP32-4BIT_DEFAULT', 'OV_FP16-4BIT_DEFAULT', 'OV_FP32-4BIT_MAXIMUM', 'OV_FP16-4BIT_MAXIMUM']
+
+
+KNOWN_FRAMEWORKS = ['pytorch', 'ov', 'dldt', 'compressed_weights']
```

Comment:
compressed_weighs is not framework.
Maybe it is better to have common list of skip model directory:

IGNORE_MODEL_PATH_PARTS = KNOWN_FRAMEWORKS + KNOWN_PRECISIONS + ["com]pressed_weights"]

Reply:
or move it to precisions (weights compression is more related to precision than framework)